abacus-integration-test/e2e/Attributes.spec.ts

```diff
@@ -222,58 +206,123 @@ test.describe('<Attributes/>', () => {
 
     await test.step('Sort by Name ASC', async () => {
-      await page.click(sortByToolbarButton)
-      await ascendingSortingOption.click()
-      await nameSortingSubOption.click()
+      await page.locator(sortByToolbarButton).click({ force: true })
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'ASC'}).click()
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'name'}).click()
       await assertItemsOrderAfterSorting(firstAttributeName, thirdAttributeName, secondAttributeName)
     })
 
     await test.step('Sort by Name DESC', async () => {
-      await page.click(sortByToolbarButton, {force: true})
-      await descendingSortingOption.click()
-      await nameSortingSubOption.click()
+      await page.locator(sortByToolbarButton).click({ force: true })
+      await page.waitForSelector('.ant-cascader-menu-item-content')
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'DES'}).click()
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'name'}).click()
       await assertItemsOrderAfterSorting(secondAttributeName, thirdAttributeName, firstAttributeName)
     })
 
     await test.step('Sort by Rule ASC', async () => {
-      await page.click(sortByToolbarButton, {force: true})
-      await ascendingSortingOption.click()
-      await ruleSortingSubOption.click()
+      await page.locator(sortByToolbarButton).click({ force: true })
+      await page.waitForSelector('.ant-cascader-menu-item-content')
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'ASC'}).click()
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'rule'}).click()
       await assertItemsOrderAfterSorting(secondAttributeName, firstAttributeName, thirdAttributeName)
     })
 
     await test.step('Sort by Rule DESC', async () => {
-      await page.click(sortByToolbarButton, {force: true})
-      await descendingSortingOption.click()
-      await ruleSortingSubOption.click()
+      await page.locator(sortByToolbarButton).click({ force: true })
+      await page.waitForSelector('.ant-cascader-menu-item-content')
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'DES'}).click()
+      await page.locator('.ant-cascader-menu-item-content', {hasText: 'rule'}).click()
       await assertItemsOrderAfterSorting(thirdAttributeName, firstAttributeName, secondAttributeName)
     })
 
+    await test.step('Cleanup', async () => {
+      await deleteAttributeViaAPI(apiContext, authority, firstAttributeName,['A', 'G', 'H'], "anyOf")
+      await deleteAttributeViaAPI(apiContext, authority, secondAttributeName,['C', 'G', 'H'], "allOf")
+      await deleteAttributeViaAPI(apiContext, authority, thirdAttributeName,['B', 'G', 'H'], "hierarchy")
+    })
+  });
+
+  test('should sort attributes by rule, values_array', async ({ page, authority}) => {
+    const sortByToolbarButton = selectors.attributesPage.attributesHeader.sortByToolbarButton
```

Comment:
lot of duplicated code after division. Do we really need it? Doesn't increasing test timeout solve time-to-time failures?